examples/file-feed/complete/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "file-feed",
```

Comment:
I've renamed it to `file-feed` as I'm afraid of stamping something with a "InterPlanetary X" name that is not a project (i.e example/tutorial) that we support and make it have the same level of activity of a product. 

Pointing people to the 'file-feed tutorial' has a completely different tone than 'InterPlanetary FileExchange'

@haadcode hope you understand, we can chat about it and reach an agreement.